test/e2e/storage/persistent_volumes-local.go

```diff
@@ -669,43 +682,47 @@ func createVolumeConfigMap(config *localTestConfig) {
 	Expect(err).NotTo(HaveOccurred())
 }
 
-func createBootstrapperPod(config *localTestConfig) {
-	pod := &v1.Pod{
+func createBootstrapperJob(config *localTestConfig) {
```

Comment:
why the switch from pod to job?

Reply:
The function of bootstrapper is just to start Daemon set with the provisioner, before with pod, it has to be manually deleted and with job it is not required.